test/tests/toolbar_toolbar.js

```diff
@@ -1115,7 +1116,8 @@ ariaTest(
   'toolbar-button-enter-or-space',
   async (t) => {
     let textarea = await t.context.session.findElement(By.css('textarea'));
-    await textarea.sendKeys(Key.chord(Key.CONTROL, 'a'));
+    let modifierKey = translatePlatformKey(Key.CONTROL);
```

Comment:
This being `modifierKey` as an array could be confusing. Renaming to `modifierKeys` works? Although maybe just as confusing given a single value was passed. I suppose this goes back to having the function provide a clear example of usage

Reply:
Good idea! I addressed with [aa905f6](https://github.com/w3c/aria-practices/pull/3115/commits/aa905f638bbea694cde78fc7b386647bdcc4ac8c).